hw/mcu/dialog/da1469x/src/da1469x_periph.c

```diff
@@ -204,6 +209,11 @@ static struct pwm_dev os_bsp_pwm1;
 static struct pwm_dev os_bsp_pwm2;
 #endif
 
+#if MYNEWT_VAL(DA1469X_CHARGER)
+struct da1469x_charger_dev da1469x_charger_dev = {
```

Comment:
Probably no need to initialize this to all zeroes. This might make this be part of .data instead of .bss. Which would be a waste.